src/components/__tests__/checker.js

```diff
@@ -184,6 +186,35 @@ describe("check", () => {
       }).not.toThrow()
     })
   })
+
+  describe("close", () => {
+    beforeEach(() => jest.useFakeTimers())
+    afterEach(() => {
+      jest.useRealTimers()
+      jest.restoreAllMocks()
+      component.unmount()
+    })
+
+    it("calls editor.on('Remove') when mounted", () => {
+      component = mount(<Checker getBody={() => node} editor={fakeEditor} />)
+      instance = component.instance()
+      instance.check() // open it
+      jest.runAllTimers()
+      expect(fakeEditor.on).toHaveBeenCalled()
+      expect(fakeEditor.on.mock.calls[0][0]).toEqual("Remove")
```

Comment:
Perhaps knocking both these assertions out with one line would be best?
```suggestion
      expect(fakeEditor.on).toHaveBeenCalledWith("Remove")
```

Reply:
That doesn't work because editor.on is called with 2 arguments, "Remove", and an anonymous function. toHaveBeenCalledWith("Remove") fails becuase the 2nd arg is undefined

Reply:
Ahh okay this is probably fine like this then :)